### PR TITLE
Update Parameter in Tutorials

### PR DESCRIPTION
**Pull request by jeff-hernandez**
Fixes [docs](https://compose.alteryx.com/en/fix_docs/) by updating automl search parameter.

### CODE REVIEW

File: docs/requirements.txt [hunk 1]
```diff
@@ -2,7 +2,8 @@ jupyter>=1.0.0
 nbconvert>=5.5.0
 nbsphinx>=0.4.2
 Sphinx>=3.0.3
-evalml>=0.13.2
+woodwork==0.0.4
```
Comment (rwedge) [inline docs/requirements.txt:5]: what made pinning woodwork necessary?

Comment (jeff-hernandez) [reply docs/requirements.txt:5]: I get an error from evalml when using the latest version of woodwork.

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-11-33cef934ebe8> in <module>
     11     y=y_train,
     12     data_checks='disabled',
---> 13     show_iteration_plot=False,
     14 )

/usr/local/Caskroom/miniconda/base/envs/3.7/lib/python3.7/site-packages/evalml/automl/automl_search.py in search(self, X, y, data_checks, show_iteration_plot)
    390 
    391         text_column_vals = X.select('natural_language')
--> 392         text_columns = list(text_column_vals.to_pandas().columns)
    393         if len(text_columns) == 0:
    394             text_columns = None

AttributeError: 'DataTable' object has no attribute 'to_pandas'
```

Comment (rwedge) [reply docs/requirements.txt:5]: Does/did EvalML have an issue about this?

Comment (jeff-hernandez) [reply docs/requirements.txt:5]: It doesn't seem like it. I'll check with the team.

Comment (jeff-hernandez) [reply docs/requirements.txt:5]: This issue is resolved in the main branch of EvalML.

**Review by rwedge — APPROVED** [review]
Looks good!

**Pull request merged**